packages/hash/api/package.json

```diff
@@ -29,9 +29,9 @@
     "@types/redis": "2.8.32",
     "ajv": "8.11.0",
     "ajv-formats": "2.1.1",
-    "apollo-datasource": "3.3.0",
-    "apollo-server-errors": "3.1.0",
-    "apollo-server-express": "2.25.3",
+    "apollo-datasource": "3.3.2",
+    "apollo-server-errors": "3.3.1",
+    "apollo-server-express": "3.9.0",
```

Comment:
Needed to bump `ts-invariant` to avoid https://github.com/apollographql/apollo-client/issues/9094#issuecomment-1176216035